v2/stacks/feedback/default.env

```diff
@@ -0,0 +1,15 @@
+# -- Global variable overrides --
+IS_PUBLISHING="false"
+MAIL_ENCRYPTION=${MAIL_ENCRYPTION:-false}
```

Comment:
Doesn't need to be here - put this in the manifest instead.

Reply:
Done